### PR TITLE
Migrate/obsolete incongruent Leadworks, EBR, and Rivtech guns

### PR DESCRIPTION
**Pull request by tenmillimaster**
#### Summary
```SUMMARY: Content "migrates+obsoletes firearms incongruent with lore"```

#### Purpose of change
Leadworks largely takes a leaf from the XM8 program, which sought to integrate modularity into a service rifle, but not at a user level. The rifles, and other similar built-in-mod guns have not seen success in real life as they are purported to have done so in-game.
The RM360 describes an entirely fictional .22 LR gatling gun, which would be available to SOT's only. Seeing as it is mostly a range toy, I doubt a serious company would build such a thing and market it.
The Mosin EBR kits have been obsoleted by the mounting furniture gunmods.

#### Describe the solution
Obsoletes or migrates items in question. Redefines LW-Feed, obsoleted, to work with Stanags instead of migrated items. Backup revolver is obsoleted, to either be later migrated to an AR pistol or a single-shot 223 PAR-1. 
460 guns migrated to a .460 1911, magazines altered to accept them.

#### Describe alternatives you've considered
Obsoleting .460 guns and adding 1911 later.
That whole leadworks western gun company I was suggesting, but that's still a lot of mental gymnastics.
Pretending that the XM8 was actually successful.

#### Testing
Loaded mainline. Loaded generic guns. Checked to make sure everything still migrated correctly. Figuring out how to handle migration/blacklisting in GG was a bit of a bear, but I managed.
premigration
postmigration

#### Additional context
Generic guns has a potential load error- I have accidently re-used the abstract item ID `pistol_revolver` for the generic guns `police revolver`. Fix to come.

### CODE REVIEW

**Comment by I-am-Erk** [comment]
I do really want to bring Rivtech back with some other items. We should look into having them on item description snippets and things

**Comment by tenmillimaster** [comment]
> I do really want to bring Rivtech back with some other items. We should look into having them on item description snippets and things

I haven't removed any Rivtech guns in their caseless families.

**Comment by I-am-Erk** [comment]
Ah good

**Comment by anothersimulacrum** [comment]
The items that no longer exist are still on blacklists in mods, and need to be removed, because it's causing the tests to fail.

**Comment by kevingranade** [comment]
I have those fixes locally, I think I'll push this and then push my fixes afterwards.

File: data/json/items/migration.json [hunk 1]
```diff
@@ -988,5 +988,215 @@
...
+    "type": "MIGRATION",
+    "replace": "masterkey"
+  },
+  {
+    "id": "lead_glm",
+    "type": "MIGRATION",
+    "replace": "m320_mod"
+  },
+  {
+    "id": "lead_stabilizer",
```
Comment (BevapDin) [inline data/json/items/migration.json]: The leadworks items appear twice in this file (this one is migrated again on line 1003).

File: data/json/items/migration.json [hunk 2]
```diff
@@ -988,5 +988,215 @@
...
+    "type": "MIGRATION",
+    "replace": "bipod"
+  },
+  {
+    "id": "lead_u_shotgun",
+    "type": "MIGRATION",
+    "replace": "masterkey"
+  },
+  {
+    "id": "lead_glm",
```
Comment (BevapDin) [inline data/json/items/migration.json]: What's worse: this item migrated to "m320" here, but to "m320_mod" at line 1055. I assume one is a gun, the other is a gunmod. So what should it be replaced into?

Comment (tenmillimaster) [reply data/json/items/migration.json]: The former is the standalone GLM, but what I always intuitively type when referring to the m320 installed in a more typical way. The latter is the correct replacement.

**Pull request merged**